AssetInformationListener/Boundary/EntityEventSns.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace AssetInformationListener.Boundary
+{
+    /// <summary>
+    /// TODO
```

Comment:
We can remove this, as we have now created the model

Reply:
It will be when the next ticket is done. This is just setting up the application shell.